src/map/utils/guildutils.cpp

```diff
@@ -72,6 +72,10 @@ void Initialize()
     }
     TPZ_DEBUG_BREAK_IF(g_PGuildShopList.size() != 0);
 
+    // Show in log day number used for guild rank items
+    int vanaDay = CVanaTime::getInstance()->getVanaTime() / (60 * 60 * 24);
+    ShowDebug(CL_CYAN"Guild Items initialized for Vanadiel day: %d\n" CL_RESET, vanaDay);
```

Comment:
Not sure if we still want to show debug logs every hour once we're quite sure it works? :thinking:

Reply:
It's every day only

Reply:
a vana day is just shy of 1 earth hr.

Reply:
Suggestion: -if- this is going to be kept even after we're sure it works, change it to ShowNotice, which will display the same but under a diff message type. we use that type for other "thing has happened" notifications in the log.

Reply:
This will only print when the server boots.

Reply:
Didn't even notice we were talking about code in an initialization function.